scripts/ci-e2e.sh

```diff
@@ -71,6 +71,11 @@ else
   export CAPM3_FROM_RELEASE="${CAPM3_FROM_RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v0.5.")}"
   export CAPM3_TO_RELEASE="${CAPM3RELEASE}"
 fi
+if [[ $CAPM3_VERSION == "v1alpha5" ]]; then
+  export PROVIDER_ID_FORMAT="metal3://{{ '{{ ds.meta_data.uuid }}' }}"
+else
+  export PROVIDER_ID_FORMAT="metal3://{{ '{{ ds.meta_data.providerid }}' }}"
+fi
```

Comment:
I would prefer to put this in environment.sh

Reply:
Thanks, moved it to environment.sh